tests/pytest_translate

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env python3
+#
+# Translate a python test to another language.
+#
+import enum
+import sys
+import os
+import argparse
+import ast
+
+sys.path.insert(0, os.path.join(os.path.dirname(
```

Comment:
Modifying `sys.path` from code is nearly always a code smell.

Reply:
What's the alternative here?  How do I import modules in another path?

Reply:
The usual Python way, by creating a package with `__init__.py`. I wouldn't put the implementation in the tests directory anyway, I'd create a new top-level directory in the repo, put all of the translation files there, and just do regular imports. 

Now, tests to the translation code could and should be located in the tests directory. And obviously the output of such translations.